docs/source/markdown/options/read-write-tmpfs.md

```diff
@@ -0,0 +1,8 @@
+####> This option file is used in:
+####>   podman create, run
+####> If you edit this file, make sure your changes
+####> are applicable to all of those.
+#### **--read-write-tmpfs**
+
+If container is running in --read-only mode, then mount a read-write tmpfs on /run, /tmp, and /var/tmp. When false, Podman the entire container has no tmpfs that are read/write unless specified on the command line.  Podman mounts /dev, /dev/mqueue, /dev/pts, /dev/shm as read only. The default is *true*
```

Comment:
>When false, Podman the entire container

Does not parse. I think `Podman` is the part that needs to go, but I'm not sure what the intention was.

Did you deliberately un-underscore the directory names? Our guidelines say:

>Strings of characters or numbers can be highlighted with `backticks`. Paths of any kind **must** be highlighted.

(which suggests that backticks are preferred, underscores are ok, but leaving them unadorned is not).